### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by kellertuer**
This improves a few things in the docs

* [x] external links to Wikipedia or ManifoldsBase or Manopt now have prefix icons
* [x] remove the 📖 prefix that was previously used for a few Wikipedia links 
* [x] `make.jl` is a bit easier to use: If the tutorials do not exist and quarto is not run, a warning is issued and they are excluded from the docs – this makes the docs again “render on default”
* [x] we should write a bit more on the about page, I especially would like to link a few packages using the library.

### CODE REVIEW

File: tutorials/getstarted.qmd [hunk 1]
```diff
@@ -318,7 +318,7 @@ The same mechanism exists for ``[🔗 `parallel_transport_to(M, p, X, q)`](@extr
...
 
-We again look at the [📖 exponential map](https://en.wikip edia.org/wiki/Exponential_map_(Riemannian_geometry)). On a manifold `M` the exponential map needs a point `p` (to start from) and a tangent vector `X`, which can be seen as direction to “walk into” as well as the length to walk into this direction. In `Manifolds.jl` the function can then be called with `q = exp(M, p, X)` (see ``[🔗 `exp(M, p, X)`](@extref `Base.exp-Tuple{AbstractManifold, Any, Any}`)``{=commonmark}). This function returns the resulting point `q`, which requires to allocate new memory.
+We again look at the [exponential map](https://en.wikip edia.org/wiki/Exponential_map_(Riemannian_geometry)). On a manifold `M` the exponential map needs a point `p` (to start from) and a tangent vector `X`, which can be seen as direction to “walk into” as well as the length to walk into this direction. In `Manifolds.jl` the function can then be called with `q = exp(M, p, X)` (see ``[🔗 `exp(M, p, X)`](@extref `Base.exp-Tuple{AbstractManifold, Any, Any}`)``{=commonmark}). This function returns the resulting point `q`, which requires to allocate new memory.
```
Comment (mateuszbaran) [inline tutorials/getstarted.qmd:321]: Should we also remove "🔗"?

Comment (kellertuer) [reply tutorials/getstarted.qmd]: Well-spotted. I totally forgot about that idea I used (but also only in this tutorial). I replaced all such occurrences.

**Comment by kellertuer** [comment]
I also started on the last point, what I still want to do (but that needs a bit of time):

* mention contributions (larger than fixing typos) by going through our PRs
* maybe mention more package that use Manifolds.jl beyond our own ones.

**Comment by kellertuer** [comment]
I went through the PRs and added contributors that contributed a manifold; added Caesar, which was the main package I could google to actually use Manifolds.jl

File: docs/src/misc/about.md [hunk 2]
```diff
@@ -1,17 +1,36 @@
 # About `Manifolds.jl`
 
-## License
-
-[MIT License](https://github.com/JuliaManifolds/Manifolds.jl/blob/master/LICENSE)
+`Manifolds.jl` was started by [Seth Axen](https://github.com/sethaxen), [Mateusz Baran](https://github.com/mateuszbaran), [Ronny Bergmann](https://github.com/kellertuer), and [Antoine Levitt](https://github.com/antoine-levitt) in 2019, after a very fruitful discussion following the release of the first version of [`Manopt.jl`](https://manoptjl.org/). The goal of `Manifolds.jl` is to provide a library of manifolds in Julia. The manifolds are implemented using the [´ManifoldsBase.jl](https:// github.com/JuliaManifolds/ManifoldsBase.jl/) interface.
```
Comment (mateuszbaran) [inline docs/src/misc/about.md:3]:
```suggestion
`Manifolds.jl` was started by [Seth Axen](https://github.com/sethaxen), [Mateusz Baran](https://github.com/mateuszbaran), [Ronny Bergmann](https://github.com/kellertuer), and [Antoine Levitt](https://github.com/antoine-levitt) in 2019, after a very fruitful discussion following the release of the first version of [`Manopt.jl`](https://manoptjl.org/). The goal of `Manifolds.jl` is to provide a library of manifolds in Julia. The manifolds are implemented using the [´ManifoldsBase.jl](https://github.com/JuliaManifolds/ManifoldsBase.jl/) interface.
```

**Review by mateuszbaran — APPROVED** [review]

**Pull request merged**